en/flight_modes/follow_me.md

```diff
@@ -41,43 +41,46 @@ The following flight precautions should be observed:
 - Give your vehicle sufficient room to stop, especially when it is moving fast.
-  Many Android devices do not update their position very frequently, and autopilot estimations of the speed and direction can be inaccurate.
-- Be ready to take manual RC control if something goes wrong when using follow me mode for the first time.
-  The accuracy of positioning is dependent on the quality of the GPS used by target system.
-  If the GPS is not accurate, this will be reflected in follow me.
+- Be ready to switch back to Position mode, etc, if something goes wrong when using follow me mode for the first time.
+  If the target system (e.g. Android Phone)'s GPS is not accurate, this can affect the behavior.
 
-## Using Follow Me with QGroundControl
+## How to use Follow Me
 
 *Follow Me* mode is supported by *QGroundControl* on ground station hardware that has a GPS module.
 The recommended configuration is a USB OTG-capable Android device with two telemetry radios.
 
 To setup *Follow Me* mode:
-- Connect a telemetry radio to your Android device and another to the vehicle (this allows positioning information to be relayed between the two radios).
+- Connect a telemetry radio to your ground station device and another to the vehicle (this allows positioning information to be relayed between the two radios).
 - Disable sleep-mode on your Android device:
   - This setting can usually be found under: **Settings \> Display**.
   - It is important that you set your Android device to not go to sleep as this could cause the GPS signal to cease being emitted at regular intervals.
 - Takeoff to a height of at least 2-3 metres (recommended even though auto-takeoff is supported).
   - Set the vehicle on the ground, press the safety switch and step back at least 10 meters.
   - Arm the vehicle and takeoff.
 - Switch into follow me mode.
-  - The copter will ascend to the specified [minimum height](#NAV_MIN_FT_HT) and then pause for a moment to assess the radio link.
-    If the link update rate is OK the multicopter will then yaw to face the target.
+  - The copter will first ascend to the specified [minimum height](#NAV_MIN_FT_HT) while facing the target, then move horizontally, to avoid potential collisions
 
-At this point you should be able to start moving and the copter should follow your movements.
+At this point you can start moving, and the drone will be following you.
 
 The mode has been tested on the following Android devices:
 - Nexus 5
 - Nexus 7 Tablet
-
+- Galaxy S10
 
 ## Configuration
 
-The follow-me behaviour can be configured using the following parameters:
+:::warning
+Setting the **Altitude mode ([NAV_FT_ALT_M](../advanced_config/parameter_reference.md#NAV_FT_ALT_M)**) to '3D Tracking' can be dangerous and unstable, as Android phones for example are known for having huge altitude errors in their GPS coordinates, around 10 to 50 meters. Therefore, 3D Tracking is only advised if you have an accurate GPS source for the Ground Station!

```

Comment:
Perhaps reorder to instruct first:
```suggestion
Do not set the **Altitude mode ([NAV_FT_ALT_M](#NAV_FT_ALT_M)**) to `3D Tracking`!
This can be dangerous and unstable because many Android phones (and other devices) are known to have significant altitude errors in their GPS coordinates (around 10 to 50 meters!). 3D Tracking is only advised if you have an accurate GPS source for the Ground Station!
```